src/models/SinglePoll.ts

```diff
@@ -0,0 +1,66 @@
+import TwitterInfo from './TwitterInfo'
+import VotesInfo from './VotesInfo'
+
+export default class SinglePoll {
+  /**
+   * UTC midnight on day of the poll
+   *
+   * @memberof SinglePoll
```

Comment:
These member ofs are not needed I think.
What **is** needed are access modifyers, one of:

- `public`
- `private` (or use `#` for actual private fields, i.e. `#private filed` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)
- `protected` (private, but accesible by subclass)